src/main/java/colress/command/AddCommand.java

```diff
@@ -0,0 +1,80 @@
+package colress.command;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+import colress.TaskList;
+import colress.TaskType;
+import colress.Ui;
+import colress.task.Deadline;
+import colress.task.Event;
+import colress.task.Task;
+import colress.task.ToDo;
+
+/**
+ * Represents the add command that add a task to the list of tasks.
+ */
+public final class AddCommand extends Command {
```

Comment:
Not sure if this is valid but at first glance, i thought that "AddCommand" sounds like a method name to add a command because it starts with a verb 😅